modules/installation-guide/partials/proc_importing-tls-certificates-to-che-server-java-truststore.adoc

```diff
@@ -1,92 +1,174 @@
 
 
 [id="importing-tls-certificates-to-{prod-id-short}-server-java-trustore_{context}"]
-= Importing TLS certificates to {prod-short} server Java truststore
+= Importing external or additional TLS certificates to {prod-short}
 
-When {prod-short} server attempts to send an HTTPS request to an external service as {identity-provider}, a proxy or a git server, the connection fails if {prod-short} does not trust the TLS certificate used by the external service.
+Network communications between the components of a {prod-short} installation and the started workspaces, are all secured through the TLS protocol, and thus require the use of trusted certificate authorities. 
 
-To fix this problem, configure {prod-short} to authorize HTTPS communication with external services, such as identity and Git servers, by adding information about the TLS certificates to the {prod-short} configuration.
+In standard installations performed with the {prod-short} operator, the operator itself takes care of setting up and propagating the required TLS certificates, and no manual step should be necessary.
+
+But in some cases it can be necessary to add TLS certificates to the {prod-short} installation, and have them used as trusted certificate authorities by {prod-short}.
```

Comment:
```suggestion
In some cases it can be necessary to add TLS certificates to the {prod-short} installation, so that every {prod-short} component will consider them as signed by a trusted CA.
```

Reply:
Fixed